server/src/routers/coordinatesRouter.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express')
+const router = new express.Router()
+
+const Coordinates = require('../models/coordinates')
+
+const coordinatesValidator = require('../middlewares/coordinatesHandlers')
+
+//The distance should be in meters from which we need to find the parking ads
+router.get('/search', coordinatesValidator, async (req, res, next) => {
+    const longitude = req.body.longitude
+    const latitude = req.body.latitude
```

Comment:
Question: Though it does not matter as part of the application on front-end, do you think an ideal situation should have latitude and longitude as query strings as this is not sensitive information?

Reply:
Yes we can have them as query strings too but I was just trying to make all the routes similar and since I have not used query string anywhere else, I kept it like that only here also. But it might make sense to convert them into query strings because that is what most websites do like Google Maps. Let me know if I should change it.

Reply:
Can you please have a look at this again?